src/Modifier/Product/VariantAttributesModifier.php

```diff
@@ -20,6 +22,8 @@
  */
 class VariantAttributesModifier extends Modifier
 {
+    use SymfonyContainerTrait;
+
```

Comment:
This mustn't be used here. It is only for overwritten OXID classes where dependency injection is not possible.

```suggestion
```